scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/vqfn.yaml

```diff
@@ -561,6 +561,50 @@ VQFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+VQFN-32-1EP_5x5mm_P0.5mm_EP3.15x3.15mm:
+  device_type: 'VQFN'
+  size_source: 'https://www.ti.com/lit/ds/slvs589d/slvs589d.pdf#page=33'
+  ipc_class: 'qfn'
+  body_size_x:
+    minimum: 4.85
+    nominal: 5
+    maximum: 5.15
+  body_size_y:
+    minimum: 4.85
+    nominal: 5
+    maximum: 5.15
+  overall_height:
+    minimum: 0.8
```

Comment:
minimum height is 0.85

Reply:
I believe it is still 0.8, this is specified on another page of the datasheet - I have added an extra screenshot to the top.

Reply:
tbh, now 3 days later I can't find that 0.85mm value anymore.  `0.8` looks good.